lib/MusicBrainz/Server/Controller/Relationship/LinkAttributeType.pm

```diff
@@ -77,24 +78,48 @@ sub edit : Chained('load') RequireAuth(relationship_editor)
 
     my $form = $c->form( form => 'Admin::LinkAttributeType', init_object => $link_attr_type );
 
+    my $attribute_in_use = $c->model('LinkAttributeType')->in_use($link_attr_type->id);
+
+    if ($attribute_in_use) {
+        $form->field('free_text')->disabled(1);
+        if ($link_attr_type->creditable) {
+            $form->field('creditable')->disabled(1);
+        }
+    }
```

Comment:
This disables the field so that it is ignored in the backend, but the checkbox still shows undisabled in the frontend. Not sure if we have a good way of changing that with TT. Since this is for admins only, it might not matter a ton, but if you have ideas, I'd love to improve it.

Reply:
It seems fixable by adding `disabled => $field->disabled,` to `_render_input` in FormRenderer. Want to push a commit for that?